### PR TITLE
Bugfix for Temperature correction

### PR DESCRIPTION
**Pull request by HolgerPollyNet**
bux fix for T correction

### CODE REVIEW

File: lib/interface/picassoProcV3.m [hunk 1]
```diff
@@ -398,7 +398,8 @@
             'flag407nmChannel', data.flag407nmChannel, ...
             'flag355nmRotRaman', data.flag355nmChannel & data.flagRotRamanChannel, ...
             'flag532nmRotRaman', data.flag532nmChannel & data.flagRotRamanChannel, ...
-            'flag1064nmRotRaman', data.flag1064nmChannel & data.flagRotRamanChannel);
+            'flag1064nmRotRaman', data.flag1064nmChannel & data.flagRotRamanChannel, ...
+            'isUseLatestGDAS', PollyConfig.flagUseLatestGDAS);
```
Comment (ZPYin) [inline lib/interface/picassoProcV3.m:402]: should this be `flagUseLatestGDAS`, according to your relavant fix in `picassoProcV3.m`?

----------------
Maybe one tip after changing the arguments of function, i.e., input/output/keywords. One should also change the docstring of the function in the headers. It's good for others to use the updated function correctly.

File: lib/preprocess/pollyPreprocess.m [hunk 2]
```diff
@@ -295,7 +296,7 @@
     
     syms T
     for iCh = 1:size(data.signal, 1)
-        corFunc = sym(config.tempCorFunc{iCh});
+	 corFunc = sym(config.tempCorFunc{iCh});
```
[the remark this answers is not in the document]
Comment (HolgerPollyNet) [reply lib/preprocess/pollyPreprocess.m:299]: no, I did not really make changes here.....obviously the encoding again

File: lib/preprocess/pollyPreprocess.m [hunk 3]
```diff
@@ -195,6 +195,7 @@
 addParameter(p, 'flag355nmRotRaman', false, @islogical);
 addParameter(p, 'flag532nmRotRaman', false, @islogical);
 addParameter(p, 'flag1064nmRotRaman', false, @islogical);
+addParameter(p, 'flagUseLatestGDAS', false, @islogical);
```
Comment (HolgerPollyNet) [inline lib/preprocess/pollyPreprocess.m:198]: this is the only thing I added

File: lib/interface/picassoProcV3.m (hunk 1 above)
Comment (HolgerPollyNet) [reply lib/interface/picassoProcV3.m:402]: Actually, I have no clue ;-) The way it is now it is working, but feel free to change it in a way you think it is more consistent....

Comment (HolgerPollyNet) [reply lib/interface/picassoProcV3.m:402]: I did not want to change line 758, and thus just copied from there....

Comment (HolgerPollyNet) [reply lib/interface/picassoProcV3.m:402]: Feel free to chnage oit for more consistency

**Review by ZPYin — APPROVED** [review]

**Pull request merged**